docs/docsite/rst/porting_guides/porting_guide_2.6.rst

```diff
@@ -62,7 +62,7 @@ Noteworthy module changes
   destination path like this::
 
     $ ansible localhost -m file -a 'path=/tmp/lib state=directory'
-
+* Openstack modules are no longer using ``shade`` library. Instead ``openstacksdk`` is used. Since ``openstacksdk`` should be already present as a dependency to ``shade`` no additional actions are required.
 
```

Comment:
Can you also add a line to say if yyou were using `contrib/inventory/openstack.py` they should now use `contrib/inventory/openstack_inventory.py`

Reply:
is "Porting custom scripts" section ok for that? Was thinking about that already, but didn't find really fitting section

Reply:
How about a new section for "Dynamic inventory scripts"?

Reply:
@gundalow this way?